autotest.pm

```diff
@@ -407,6 +410,7 @@ sub runalltests {
             $t->save_test_result();
         }
     }
+    bmwqemu::save_json_file(\%modules_spent_time, "modules_spent_time.json") if (%modules_spent_time);
```

Comment:
does this save a json result file an additional time? I assume the result file is already saved in another location. Would be good to not save an additional time when this is not necessary.

Reply:
yes, those spent times will be saved in a json file. Do you mean I should save the spent time in the `detail-xxxx.json` file?

Reply:
Yes, but it is a little bit complicated, see https://github.com/os-autoinst/openQA/pull/2629#pullrequestreview-339219736.